cumulus/parachains/integration-tests/emulated/assets/asset-hub-westend/src/tests/set_xcm_versions.rs

```diff
@@ -47,32 +47,21 @@ fn relay_sets_system_para_xcm_supported_version() {
 #[test]
 fn system_para_sets_relay_xcm_supported_version() {
 	// Init test variables
-	let sudo_origin = <Westend as Chain>::RuntimeOrigin::root();
 	let parent_location = AssetHubWestend::parent_location();
-	let system_para_destination: VersionedMultiLocation =
-		Westend::child_location_of(AssetHubWestend::para_id()).into();
-	let call = <AssetHubWestend as Chain>::RuntimeCall::PolkadotXcm(pallet_xcm::Call::<
-		<AssetHubWestend as Chain>::Runtime,
-	>::force_xcm_version {
-		location: bx!(parent_location),
-		version: XCM_V3,
-	})
-	.encode()
-	.into();
-	let origin_kind = OriginKind::Superuser;
-
-	let xcm = xcm_transact_unpaid_execution(call, origin_kind);
-
-	// System Parachain sets supported version for Relay Chain throught it
-	Westend::execute_with(|| {
-		assert_ok!(<Westend as WestendPallet>::XcmPallet::send(
-			sudo_origin,
-			bx!(system_para_destination),
-			bx!(xcm),
-		));
 
-		Westend::assert_xcm_pallet_sent();
-	});
+	// System Parachain sets supported version for Relay Chain through it
+	Westend::send_transact_to_parachain(
+		OriginKind::Superuser,
+		AssetHubWestend::para_id(),
+		<AssetHubWestend as Chain>::RuntimeCall::PolkadotXcm(pallet_xcm::Call::<
+			<AssetHubWestend as Chain>::Runtime,
+		>::force_xcm_version {
+			location: bx!(parent_location),
+			version: XCM_V3,
+		})
```

Comment:
I would move this to its own variable (`let call = ...`) just to make it more readable and easy to understand what's happening

Reply:
fixed